component/prometheus/exporter/mssql/mssql.go

```diff
@@ -36,10 +39,12 @@ var DefaultArguments = Arguments{
 
 // Arguments controls the mssql exporter.
 type Arguments struct {
-	ConnectionString   rivertypes.Secret `river:"connection_string,attr"`
-	MaxIdleConnections int               `river:"max_idle_connections,attr,optional"`
-	MaxOpenConnections int               `river:"max_open_connections,attr,optional"`
-	Timeout            time.Duration     `river:"timeout,attr,optional"`
+	ConnectionString   rivertypes.Secret         `river:"connection_string,attr"`
+	MaxIdleConnections int                       `river:"max_idle_connections,attr,optional"`
+	MaxOpenConnections int                       `river:"max_open_connections,attr,optional"`
+	Timeout            time.Duration             `river:"timeout,attr,optional"`
+	QueryConfigFile    string                    `river:"query_config_file,attr,optional"`
```

Comment:
I would remove the query_config_file entirely.

Reply:
@mattdurham I can definitely do this, but any particular reason why (the existing pattern seems to have both)?

Reply:
I think the only other place we do this is with TLS which we carried over existing behavior. The main reason I am not a fan is that there are two ways of doing something and we have to define how each one works. If we want both we need to add logic to return an error if both are set below when creating the collector. Currently if user selects both then the file takes precedent but that's not relayed to the user.

Reply:
@mattdurham I removed this functionality. Now only query_config is available.